README.md

```diff
@@ -116,15 +115,39 @@ kubectl apply -f manifests/logging-stack
 # kubectl kustomize manifests/logging-stack/ | kubectl apply -f -
 ```
 
+> You also can deploy Fluent Bit Operator with Helm chart.
+> Note: For the Helm-based installation you need Helm v3.2.1 or later.
+
+Install the version without the plug-in
+
+```
+helm install fluentbit-operator  --create-namespace -n kubesphere-logging-system chart/fluentbit-operator/charts/k8s/
+```
+Install the version with the plug-in
+
+```
+helm install fluentbit-operator  --create-namespace -n kubesphere-logging-system chart/fluentbit-operator/
+```
+
 Within a couple of minutes, you should observe an index available:
 
 ```shell
 $ curl localhost:9200/_cat/indices
 green open ks-logstash-log-2020.04.26 uwQuoO90TwyigqYRW7MDYQ 1 1  99937 0  31.2mb  31.2mb
-``` 
+```
 
 Success!
 
+### Collect containerd logs
```

Comment:
It needs to support the installation of Fluent Bit Operator on `containerd` with helm.